browser/src/components/core/AutoGrowInput.tsx

```diff
@@ -45,18 +46,25 @@ export function AutoGrowInput({
     //   >
     //     {value}
     //   </span>
-    <div className={`grow-wrap ${className ? className : ""}`}>
-      {/* <div className={`grow-wrap`}> */}
-      <textarea
-        {...(extraProps || {})}
-        className="form-textarea mt-1 block w-full"
-        style={{ width: "300px", minHeight: "200px" }}
-        // placeholder="Enter your response to the prompt..."
-        value={value}
-        onChange={(event) => onChange(event.target.value)}
-      ></textarea>
-    </div>
-
+    <>
+      <div className={`grow-wrap ${className ? className : ""}`}>
+        {/* <div className={`grow-wrap`}>
+        <textarea
+          {...(extraProps || {})}
+          className="form-textarea mt-1 block w-full"
+          style={{ width: "300px", minHeight: "200px" }}
+          // placeholder="Enter your response to the prompt..."
+          // value={value}
+          // onChange={(event) => onChange(event.target.value)}
+        >
+        </textarea> */}
+        <Editor
+          value={value} 
+          onChange={onChange}
+          placeholder={extraProps.placeholder ? extraProps.placeholder : null} 
+        />
+      </div>
```

Comment:
oh also i added a maxLength on the text area in a recent change (you'll need to merge/rebase with main) and that would be good to incorporate here (900 characters)

Reply:
oh nice! just updated to support this using tiptap's character count